src/test/assembly/x86_64-floating-point-clamp.rs

```diff
@@ -0,0 +1,25 @@
+// Floating-point clamp is designed to be implementable as max+min,
+// so check to make sure that's what it's actually emitting.
+
+// assembly-output: emit-asm
+// compile-flags: -O -C "llvm-args=-x86-asm-syntax=intel"
```

Comment:
```text
--- stderr -------------------------------
error: unknown codegen option: `"llvm-args`
------------------------------------------
```

Oh, looks like I gave bad advice here.  Maybe try this:

```suggestion
// compile-flags: -O -C llvm-args=-x86-asm-syntax=intel
```